src/backend/gdb_expansion.ts

```diff
@@ -231,6 +247,15 @@ export function expandValue(variableCreate: (arg: VariableObject | string, optio
 			ref = variableCreate(getNamespace(name));
 			val = "...";
 		}
+		value = value.trim();
+		if (value[0] == ','){
+			let tmp = value;
+			tmp =  tmp.substring(1).trim();
+			if(tmp.startsWith("<No data fields>")){
+				value = tmp = tmp.substring("<No data fields>".length);
+			}
```

Comment:
keeping the style would mean using `value` instead of `tmp` and for the second check just doing
```ts
value.replace(/^<No data fields>/, "");
```

Reply:
> keeping the style would mean using `value` instead of `tmp` and for the second check just doing
> 
> ```ts
> value.replace(/^<No data fields>/, "");
> ```

 Thank you for your suggestion, the code has been modified and unit tests have been added.